apps/financial-aid/web-osk/src/utils/useFormNavigation.ts

```diff
@@ -11,32 +11,35 @@ interface NavigationInfoProps {
   nextUrl?: string
 }
 
-const useFormNavigation = (currentRoute: string): NavigationInfoProps => {
-  const { form, updateForm } = useContext(FormContext)
-
-  var navigationTree = useNavigationTree(Boolean(form?.hasIncome))
-
-  const findSectionIndex = (navigationTree: FormStepperSection[]) => {
-    for (const [index, item] of Object.entries(navigationTree)) {
-      if (item.url === currentRoute) {
-        return { activeSectionIndex: parseInt(index) }
-      }
+export const findSectionIndex = (
```

Comment:
Should this live somewhere else?

Reply:
Great question! I though about it, was not sure, maybe it should be its own file? I kept it there because it is used there also, what do you think? 🤔 💭